devito/passes/clusters/blocking.py

```diff
@@ -184,6 +184,10 @@ def callback(self, clusters, prefix):
             return clusters
 
         d = prefix[-1].dim
+        if d.is_Default:
```

Comment:
Is this related? 

Not sure about the "small" but I get the idea and makes code nicer avoid weird blocks so I like it.

Reply:
no unrelated sorry forgot to mention!

Reply:
Ok tanks. Still happy with it, GTG to be merged once green.